tests/test_config_consistency.py

```diff
@@ -544,11 +544,11 @@ def test_log_injection_default(self):
 @scenarios.tracing_config_nondefault
 @features.log_injection
 @features.log_injection_128bit_traceid
-class Test_Config_LogInjection_128Bit_TraceId_Default:
+class Test_Config_LogInjection_128Bit_TraceId_Enabled:
```

Comment:
Isn't the point of "Default" to show what the feature's status by default -- in this case, the feature is trace ID format used in the logs [when log injection is enabled] and its default status is enabled. Hence the original name of the test.
I would keep the original name instead, but if you feel strongly about the change, maybe just change the comment below to something like, """Verify trace IDs are logged in 128bit format **by default** when log injection is enabled"""

Reply:
This one is bit tricky. Log injection is disabled by default. Which is why this test case uses tracing_config_nondefault scenario. I agree that the new name is equally confusing since it hides the initial intent (test log injection when the 128 bit logging config is unset). I think we can communicate this intent through your doc string suggestion.